app/assets/stylesheets/application.css.scss

```diff
@@ -22,7 +24,7 @@ body {
 }
 
 body h1, h2, h3, h4, h5, h6, .h1, .h2, .h3, .h4, .h5, .h6 {
-  font-family: monospace ;
+  font-family: 'Titillium Web', sans-serif;
```

Comment:
Properties should be ordered color, font-family, font-weight, line-height<br>Each selector in a comma sequence should be on its own line